packages/eslint-config/index.js

```diff
@@ -0,0 +1,6 @@
+// Copyright IBM Corp. 2019. All Rights Reserved.
+// Node module: @loopback/eslint-config
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+module.exports = require('./.eslintrc');
```

Comment:
I find this rather ugly. Can we use `main` property in `package.json` to specify the file providing the configuration?